Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/164582
+@@||proxima.shopimind.com^|
```

Comment:
I suggest to exclude a blocking rule, and block tracking domains in our TPF.
* `||app.shopimind.com^$third-party` (https://www.imagesdepinal.com/)
* `||iss.shopimind.com^$third-party` (https://www.drone-fpv-racer.com/)